test/geocoders/arcgis.py

```diff
@@ -98,3 +105,29 @@ def test_basic_address(self):
             {"query": "Potsdamer Platz, Berlin, Deutschland"},
             {"latitude": 52.5094982, "longitude": 13.3765983},
         )
+        
+        
+@unittest.skipUnless(  # pylint: disable=R0904,C0111
+    env.get('ARCGIS_USERNAME') is not None \
```

Comment:
I believe this condition should refer to `ARCGIS_CLIENT_ID` and `ARCGIS_CLIENT_SECRET` instead of `ARCGIS_USERNAME`/etc.